sb_logger_util/index.js

```diff
@@ -3,70 +3,72 @@
  * @author: Anuj Gupta
  * @desc: using log4s, enables application wide logging.
  */
-var log4js = require('log4js');
-var fs = require('fs');
-var dir = '../../log';
+var log4js = require('log4js')
+var fs = require('fs')
+var dir = '../../log'
+var configUtil = require('sb-config-util')
 
 if (!fs.existsSync(dir)) {
-    fs.mkdirSync(dir);
+  fs.mkdirSync(dir)
 }
 
 var options = {
-    "appenders": [{
-        "type": "clustered",
-        "appenders": [
-            {
-                "type": "console"
-            }, {
-                "type": "logLevelFilter",
-                "level": "TRACE",
-                "category": "api",
-                "appender": {
-                    "type": "file",
-                    "maxLogSize": 26214400,
-                    "filename": "../../log/microService.log"
-                }
-            }
-        ]
-    }]
-};
+  'appenders': [{
+    'type': 'clustered',
+    'appenders': [
+      {
+        'type': 'console'
+      }, {
+        'type': 'logLevelFilter',
+        'level': 'TRACE',
+        'category': 'api',
+        'appender': {
+          'type': 'file',
+          'maxLogSize': 26214400,
+          'filename': '../../log/microService.log'
+        }
+      }
+    ]
+  }]
+}
 
-log4js.configure(options);
-var logger = log4js.getLogger('api');
+log4js.configure(options)
+var logger = log4js.getLogger('api')
 
-var info = function (data) {
-    if (data) {
-        logger.setLevel('INFO');
-        logger.info(JSON.stringify(data));
-    }
+var isLoggerEnabled = function () {
+  var loggerEnabled = configUtil.getConfig('SERVICE_LOGGER_ENABLED')
```

Comment:
**SERVICE_LOGGER_ENABLED** can be **ENABLE_LOGGING** and rename env to **sunbird_content_service_enable_logging**